pyads/structs.py

```diff
@@ -349,17 +353,17 @@ def _get_string(self, offset, length):
 
     @property
     def name(self):
```

Comment:
I just see that we don't have any type annotations for the whole module. I need to take care of this. If you like you can already add type annotations to your new properties.

Reply:
I've added a  few